src/components/Reserves.js

```diff
@@ -96,6 +100,15 @@ class Reserves extends React.Component {
     this.props.history.push(`/cr/reserves/${row.courseListingId}/0/${row.id}/${row.itemId}/edit${this.props.location.search}`);
   }
 
+  getRowURL(row) {
```

Comment:
pedantic overthinking: This function configures a link to a reserve. Practically, of course, it doesn't matter whether you use names like `getRowURL(row)` or `getReserveURL(reserve)`, but the way you think about it is important. this hierarchy goes only one way; an MCL container knows about its rows and a row about its record, but the reverse is not true: a record doesn't know if it's being displayed in a row or a list or a headline. 

You don't need to change anything here, it's not worth the effort. But as you build new things, being deliberate about this kind of organization can have a clarifying effect on the rest of your code. When you are particular about the properties of a reserve, of a row containing a reserve, of a table containing a list of rows, then it becomes very obvious where to add a new function when you are tinkering, or where to look for the source of trouble when you are debugging.